src/Entity/Adherent.php

```diff
@@ -281,7 +281,7 @@ public static function create(
         string $registeredAt = 'now',
         ?array $tags = [],
         ?array $referentTags = [],
-        ?string $mandate = null
+        ?array $mandate = null
```

Comment:
IMO we can now rename `$mandate` to `$mandates` , when we have an `simple_array`, we can pluralize the property.

Reply:
:heavy_check_mark: